config/environments/production.rb

```diff
@@ -50,9 +48,15 @@
   # Use a different cache store in production.
   # config.cache_store = :mem_cache_store
 
-  # Use a real queuing backend for Active Job (and separate queues per environment)
+  # Use a real queuing backend for Active Job (and separate queues per environment).
   # config.active_job.queue_adapter     = :resque
-  # config.active_job.queue_name_prefix = "content-data-admin_#{Rails.env}"
+  # config.active_job.queue_name_prefix = "content_data_admin_production"
+
+  config.action_mailer.perform_caching = false
```

Comment:
I think we're trying to avoid changing configuration in production.rb, just checking that you'd seen this and intended it 😄